public/app/features/plugins/extensions/utils.tsx

```diff
@@ -42,28 +44,38 @@ export function handleErrorsInFn(fn: Function, errorMessagePrefix = '') {
 
 // Event helpers are designed to make it easier to trigger "core actions" from an extension event handler, e.g. opening a modal or showing a notification.
 export function getEventHelpers(context?: Readonly<object>): PluginExtensionEventHelpers {
-  const openModal: PluginExtensionEventHelpers['openModal'] = ({ title, body }) => {
-    appEvents.publish(new ShowModalReactEvent({ component: getModalWrapper({ title, body }) }));
+  const openModal: PluginExtensionEventHelpers['openModal'] = (options) => {
+    const { title, body, width, height } = options;
+
+    appEvents.publish(
+      new ShowModalReactEvent({
+        component: getModalWrapper({ title, body, width, height }),
+      })
+    );
   };
 
   return { openModal, context };
 }
 
-export type ModalWrapperProps = {
+type ModalWrapperProps = {
   onDismiss: () => void;
 };
 
 // Wraps a component with a modal.
 // This way we can make sure that the modal is closable, and we also make the usage simpler.
-export const getModalWrapper = ({
+const getModalWrapper = ({
   // The title of the modal (appears in the header)
   title,
   // A component that serves the body of the modal
   body: Body,
-}: Parameters<PluginExtensionEventHelpers['openModal']>[0]) => {
+  width,
+  height,
+}: PluginExtensionOpenModalOptions) => {
+  const className = css({ width, height });
```

Comment:
what happens if these variables are undefined? (not passed as parameters)

Reply:
I can add an assertion for that in one of the tests but they will be ignored. In this case they will create an empty style block.